### DISCUSSION TITLE
RiscOS Support?

### DISCUSSION THREAD

**Discussion by mousethatischarles** [Alex313031/thorium#913] [General]
I know, a seriously stupid request... but my favorite browser on my all-time favorite OS would be nothing short of miraculous...
...and as the fastest-loading and running usable OS supported by the Raspberry Pi foundation that also has a tiny footprint a midern browser with a similar ethos would be a good fit.

**Comment by Alex313031** [comment]
@mousethatischarles Chromium does not exist for RiscOS at all. I looked into it.

Porting Chromium to a new OS is a massive undertaking.

As an example, porting it to BSD in 2015 took many tens of people many weeks of full-time-job work.

Porting Chromium to Fuchsia STILL is not completely done, and it has been 4 years.

Assuming I even had the time, that is very flattering of you to think I would be a smart enough dev to get Chromium to work on a whole new OS that I haven't even used beyond experimenting with it back in 2022 when I got my first Raspberry Pi and wanted to try all the OSes I could try on it

Not a stupid request at all, not by a long shot. We NEED better browsers that are actually available for RiscOS. But...uhhh....probably not gonna happen, at least not from me. If someone adds RiscOS support upstream, or in a fork/patch, I would be glad to add it to Thorium.

**Comment by gz83** [comment]
This is probably not possible at the moment.
